lib/src/executable.dart

```diff
@@ -130,8 +130,7 @@ void main(List<String> args) async {
 ''';
 }
 
-Future<void> runWithConfig(
-    List<String> args, _ConfigGetter configGetter) async {
+Future<int> runWithConfig(List<String> args, _ConfigGetter configGetter) async {
```

Comment:
this was returning the exitcode before, but only if there was a handled exception, so I changed the return type to in and added a final return 0 for success.